### ISSUE TITLE
Alternative for Dependency Check

### ISSUE THREAD

**Issue by mathuriga** [DependencyTrack/dependency-track#417]
Does Dependency Track 3.5.1 use embedded Dependency-Check? ( [Removing embedded Dependency-Check and its configuration](https://github.com/DependencyTrack/dependency-track/commit/7ce10579342d637d3747fa9107606e2e722857ec) belongs to 3.5.1 )

As per milestone plan for 3.6.0, [Disable Dependency-Check Scan Ingestion Support by Default](https://github.com/DependencyTrack/dependency-track/issues/397), Dependency Track 3.6.0 won't support Dependency-Check. Is my understanding right? If so what will be the alternative will  Dependency Track use for Known Vulnerability Detection other than OSS Index for java third party dependencies?

**Comment by stevespringett** [comment]
> Does Dependency Track 3.5.1 use embedded Dependency-Check? 

Yes. DT 3.5.1 and previous used an embedded Dependency-Check instance for fuzzy NVD matching of vulnerabilities. DT also supports the ingestion of Dependency-Check scan results. These are two completely independent features.

> what will be the alternative will Dependency Track use for Known Vulnerability Detection other than OSS Index for java third party dependencies?

DT 3.6.0 will no longer embed Dependency-Check. Replacing it is a new CPE Analyzer which is more configurable and precise and will not emit false positives by default. It's also much faster. There was a screenshot of the config in the Slack channel. The replacement of Dependency-Check for analyzing CPEs is about 60% complete (in the master branch).

DT 3.6.0 will still be able to import Dependency-Check reports, but the feature will be disabled by default. Importing Dependency-Check reports will likely be removed in v3.7.0 (TBD).

**Comment by mathuriga** [comment]
@stevespringett ,

Thanks for the valuable information. The plan sounds good. When we can expect Dependency-Track 3.6.0 release?

**Comment by stevespringett** [comment]
I'm hoping to have 3.6 released by October 1st. You can follow https://github.com/DependencyTrack/dependency-track/milestone/10 for progress on 3.6.
